src/index.html

```diff
@@ -5,9 +5,45 @@
     <meta name="viewport"
           content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <title>Product cards</title>
-    <link rel="stylesheet" href="./style.css">
+    <link rel="stylesheet" href="./style.scss">
   </head>
   <body>
-    <h1>Product cards</h1>
+    <div data-qa="card" class="card">
+      <img
+      class="card__img"
+      src="../src/images/imac.jpeg"
+      alt="apple imac">
+      <div class="card__title">
+        <h2 class="card__text">APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)</h2>
+        <p class="card__code">Product code: 195434</p>
+      </div>
+      <div class="card__stars">
+        <div class="stars stars--4">
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+        </div>
+        <div class="card__reviews">
+          <span>Reviews: 5</span>
+        </div>
```

Comment:
```suggestion
        <p class="card__reviews">
          Reviews: 5
        </p>
```
you fixed it only in the place it was marked in one of the previous reviews. But generally - if there's some comment about an issue you should fix all similar cases.

In this case: `span` is not a semantic tag and such a wrapper is redundant. Just use 1 correct tag (it's not obviously `p`)